insert.go

```diff
@@ -57,13 +58,32 @@ func (b *InsertStmt) Build(d Dialect, buf Buffer) error {
 		buf.WriteValue(tuple...)
 	}
 
+	if len(b.UpsertValue) > 0 {
+		buf.WriteString(" ON DUPLICATE KEY UPDATE ")
+
+		i := 0
+		for col, v := range b.UpsertValue {
+			if i > 0 {
+				buf.WriteString(", ")
+			}
+			buf.WriteString(d.QuoteIdent(col))
+			buf.WriteString(" = ")
```

Comment:
@iktakahiro I implemented onConflict statement for mysql and postgresql driver. 
sqlite3 does not support this statement. You can find out code here: https://github.com/bgaifullin/dbr/tree/feature_on_duplicate